src/App.js

```diff
@@ -9,7 +10,14 @@ const article = {
 };
 
 const App = () => (
-  <h1>React Article</h1>
+  <>
+    <h1>React Article</h1>
+    <Article
+      title={article.title}
+      text={article.text}
+      date={article.date}
```

Comment:
```suggestion
      { ...article }
```
The way you wrote it also works. But here is a shorter way to write it